gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/internal/OrcaService.groovy

```diff
@@ -66,9 +66,9 @@ interface OrcaService {
   List getLatestExecutionsByConfigIds(@Query("pipelineConfigIds") String pipelineConfigIds, @Query("limit") Integer limit, @Query("statuses") String statuses)
 
   @Headers("Accept: application/json")
-  @GET("/applications/{application}/pipelines/{triggerType}")
+  @GET("/applications/{application}/pipelines/search")
   List searchForPipelineExecutions(@Path("application") String application,
-                                   @Path("triggerType") String triggerType,
+                                   @Query("triggerTypes") String triggerTypes,
```

Comment:
Any reason not to use a List type?

Reply:
I am following the convention used by [another existing endpoint for querying pipeline executions](https://github.com/joonlim/gate/blob/bdeea4d757b9360a1afc41c5bcc12d382dd789fb/gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ExecutionsController.java#L43) but I am open to using a List type.